server/src/utils/methods/CustomAuthChecker.ts

```diff
@@ -0,0 +1,19 @@
+// import { Context } from "apollo-server-core";
```

Comment:
Same here..

Reply:
> Remove the comments

@reynan-dev OMG so sorry about that... I'll remove it